composer.json

```diff
@@ -35,7 +35,7 @@
         "friendsofsymfony/http-cache-bundle": "^2.8",
         "handcraftedinthealps/zendsearch": "^2.0",
         "jackalope/jackalope-doctrine-dbal": "^1.3",
-        "sulu/sulu": "~2.1.9",
+        "sulu/sulu": "~2.1.9 || 2.1.*@dev",
```

Comment:
how should we handle this? i guess we need to remember to reset this before releasing the next version? 🤔